### ISSUE TITLE
from-289-retrofit-generate-loop-for-logits

### ISSUE THREAD

**Issue by david-thrower** [david-thrower/cerebros-core-algorithm-alpha#293]
# From #289 

# The issue

The composite model returns logits, whereas the old one returned probs.

# The correction to be applied 

Remove conversion from probs to logits.
